Changes

```diff
@@ -278,6 +278,11 @@ Working version
   instead of `value`.
   (Xavier Leroy, review by David Allsopp)
 
+- #10885: On Windows, change the inheritance bit (close-on-exec /
+  keep-on-exec) on sockets returned by `accept` only if needed, and
+  duplicate the socket instead of calling `SetHandleInformation`.
```

Comment:
The comment is not fully accurate: we are still calling `SetHandleInformation`, but we call it in the duplicated socket, instead of on the socket returned by `accept`.